src/index.jsx

```diff
@@ -52,7 +52,7 @@ subscribe(APP_READY, () => {
           <Route path={ROUTES.Detail.EDIT} element={<LibraryEditPage />} />
           <Route path={ROUTES.Detail.ACCESS} element={<LibraryAccessPage />} />
           <Route path={ROUTES.Detail.IMPORT} element={<CourseImportPage />} />
-          <Route path={`${ROUTES.Block.HOME}/*`} element={<LibraryBlockPage />} />
+          <Route path={`${ROUTES.Block.HOME}`} element={<LibraryBlockPage />} />
```

Comment:
don't revert this one, this route has [child routes](https://github.com/openedx/frontend-app-library-authoring/blob/master/src/library-authoring/edit-block/LibraryBlockPage.jsx#L262)

Reply:
@ormsbee any chance you could test this out on Tutor with the wildcard in this route?

Reply:
Sure.

Reply:
I actually get the same error either way. Not sure if I'm doing something wrong. 😦 

```
react.development.js:220 Warning: Failed prop type: The prop `library` is marked as required in `StudioHeaderWrapperBase`, but its value is `null`.
    at StudioHeaderWrapperBase (webpack-internal:///./src/library-authoring/studio-header-wrapper/StudioHeaderWrapper.jsx:53:7)
    at ShimmedIntlComponent (webpack-internal:///./node_modules/@edx/frontend-platform/i18n/injectIntlWithShim.js:151:7)
    at injectIntl(ShimmedIntlComponent)
    at ConnectFunction (webpack-internal:///./node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at RenderedRoute (webpack-internal:///./node_modules/react-router/dist/index.js:554:5)
    at Routes (webpack-internal:///./node_modules/react-router/dist/index.js:1185:5)
    at Router (webpack-internal:///./node_modules/react-router/dist/index.js:1123:15)
    at BrowserRouter (webpack-internal:///./node_modules/react-router-dom/dist/index.js:399:5)
    at Provider (webpack-internal:///./node_modules/react-redux/es/components/Provider.js:18:20)
    at OptionalReduxProvider (webpack-internal:///./node_modules/@edx/frontend-platform/react/OptionalReduxProvider.js:18:20)
    at ErrorBoundary (webpack-internal:///./node_modules/@edx/frontend-platform/react/ErrorBoundary.js:139:5)
    at IntlProvider (webpack-internal:///./node_modules/react-intl/lib/src/components/provider.js:92:47)
    at AppProvider (webpack-internal:///./node_modules/@edx/frontend-platform/react/AppProvider.js:108:20)
```

Reply:
Okay, so that is a warning, and if I hardcode the path for LibraryListPage to be "library-authoring", it does render.

Reply:
I'm not sure. It just seems very odd that react-router would overwrite that instead of appending.

Based on the change in how links are handled I thought it might be in https://github.com/openedx/frontend-app-library-authoring/blob/master/src/library-authoring/utils/hoc.jsx but nothing there stood out to me.

Maybe @arbrandes has some insight?

Reply:
I don't, but perhaps @Syed-Ali-Abbas-Zaidi has sufficient context.  The router-v6 changes were [pretty extensive](https://github.com/openedx/frontend-app-library-authoring/pull/197/files)!

Reply:
Questions:
Did you reinstall/rebuild after making the changes?
Just to confirm, we never tested the problem as fixed on your local tutor?

Reply:
I did restart/rebuild between changes. Going back to the commit before the react-router upgrade (so commit 16414ffb) works. Grabbing the latest master fails. Grabbing this branch also fails in the same way master is failing.